CHANGELOG.md

```diff
@@ -6,15 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
-- **BREAKING** Changed interface to:
-  - LineRipple: foundation -> lineRippleFoundation
-  - CharCounter: foundation -> charCounterFoundation
-  - FloatingLabel: foundation -> floatingLabelFoundation
-- **BREAKING** added following properties to global `Element` interface
-  - lineRippleFoundation?: MDCLineRippleFoundation
-  - charCounterFoundation?: MDCTextFieldCharacterCounterFoundation
-  - floatingLabelFoundationtion?: MDCFloatingLabelFoundation
-- internal fixes to textfield's directives and removed implicit any's
+- textfield
+  - **BREAKING** added following properties to global `Element` interface
```

Comment:
I still don't really think this is a breaking change. Nobody other than us should care about these properties. We should eventually try to get rid of the gross global type augmentation (maybe file an issue), but this doesn't seem like anything a user needs to know about.